python/lsst/pipe/tasks/characterizeImage.py

```diff
@@ -307,29 +306,39 @@ def __init__(self, butler=None, refObjLoader=None, schema=None, **kwargs):
 
     @pipeBase.timeMethod
     def runDataRef(self, dataRef, exposure=None, background=None, doUnpersist=True):
-        """!Characterize a science image and, if wanted, persist the results
+        """Characterize a science image and, if wanted, persist the results
 
         This simply unpacks the exposure and passes it to the characterize method to do the work.
 
-        @param[in] dataRef: butler data reference for science exposure
-        @param[in,out] exposure  exposure to characterize (an lsst.afw.image.ExposureF or similar).
+        Parameters
+        ----------
+        dataRef: butler data reference for science exposure
+        exposure :
+            exposure to characterize (an lsst.afw.image.ExposureF or similar).
             If None then unpersist from "postISRCCD".
             The following changes are made, depending on the config:
+
             - set psf to the measured PSF
             - set apCorrMap to the measured aperture correction
             - subtract background
             - interpolate over cosmic rays
             - update detection and cosmic ray mask planes
-        @param[in,out] background  initial model of background already subtracted from exposure
+
+        background : `lsst.afw.math.BackgroundList`
+            initial model of background already subtracted from exposure
             (an lsst.afw.math.BackgroundList). May be None if no background has been subtracted,
             which is typical for image characterization.
             A refined background model is output.
-        @param[in] doUnpersist  if True the exposure is read from the repository
+        doUnpersist :
+            if True the exposure is read from the repository
             and the exposure and background arguments must be None;
             if False the exposure must be provided.
             True is intended for running as a command-line task, False for running as a subtask
 
-        @return same data as the characterize method
+        Returns
+        -------
+        charRes :
+            same data as the characterize method
```

Comment:
This is not so helpful of a description, either link to the Characterize method or describe it again here.